dartdoc_test/lib/src/analyzer.dart

```diff
@@ -18,31 +18,31 @@ import 'package:analyzer/error/error.dart';
 import 'package:dartdoc_test/src/resource.dart';
 import 'package:source_span/source_span.dart';
 
+import 'logger.dart';
+
 void getAnalysisResult(
   AnalysisContext context,
   CodeSampleFile file,
 ) async {
   final result = await context.currentSession.getErrors(file.path);
   if (result is ErrorsResult) {
+    if (result.errors.isEmpty) {
+      log('No issues found!');
+      return;
+    }
     for (final e in result.errors) {
-      if (e.errorCode.type != ErrorType.SYNTACTIC_ERROR &&
-          e.errorCode.type != ErrorType.COMPILE_TIME_ERROR) {
-        continue;
-      }
       final span = toOriginalFileSpanFromSampleError(file, e);
       if (span != null) {
-        print(span.message((e.message)));
-        print('\n');
+        log(span.message((e.message)));
+        log('\n');
       } else {
-        print('this error is caused by generated code.');
-        print(e.message);
-        print('\n');
+        log(
+          'this error is caused by generated code.\n'
+          '$e\n',
+          LogLevel.debug,
```

Comment:
Why have we used `LogLevel.debug` specifically for this? If its not run in verbose mode and there is a problem generating `span` will we not show, the actual analysis errors?